### DISCUSSION TITLE
Can I utilize multiple GPUs for training?

### DISCUSSION THREAD

**Discussion by min731** [urchade/GLiNER#103] [Q&A] [answered]
Thank you for sharing the GLiNER open source code!

I am currently fine-tuning it in Korean, is it possible to utilize more than one GPU for training?

**Comment by urchade** [comment, marked as answer]
hi @min731, thank you for your interest in GLiNER. for new there is no support for muli-gpu

but @Ingvarstep is current working on that

**Reply by min731** [reply]
Thanks for the answer!

I am currently fine-tuning in Korean, but the model size is not large, so a single GPU is enough! Thank you!
